src/transformers/generation/configuration_utils.py

```diff
@@ -312,6 +312,10 @@ def __init__(self, **kwargs):
         self.num_assistant_tokens = kwargs.pop("num_assistant_tokens", 5)
         self.num_assistant_tokens_schedule = kwargs.pop("num_assistant_tokens_schedule", "heuristic")
 
+        # Prompt lookup decoding
+        self.prompt_lookup_num_tokens = kwargs.pop("prompt_lookup_num_tokens", 10)
```

Comment:
Attributes in this class should default to `None` whenever possible (e.g. in the lines above they are not `None` for legacy reasons)